packages/editor/src/utils/isMacOS.ts

```diff
@@ -0,0 +1,3 @@
+export const isMacOS = () => {
```

Comment:
hier klaagt sonar ook over
Unexpected negated condition.
Negated conditions should be avoided when an else clause is present [typescript:S7735](https://sonarcloud.io/organizations/nl-design-system/rules?open=typescript%3AS7735&rule_key=typescript%3AS7735)

```suggestion
export const isMacOS = () => {
  if (typeof navigator === 'undefined') return false;
  return /Mac/.test(navigator.userAgent);
};
```